workers/tests/integrationtests/test_geodatabase.py

```diff
@@ -139,6 +139,30 @@ def test_map_fid_to_uid(self):
         )
         self.assertEqual(result, "3")
 
+    def test_get_shdi_single_intersection(self):
+        """Input geometry intersects only with one SHDI region."""
+        shdi = asyncio.run(db_client.get_shdi(self.feature.geometry))
+        self.assertIsInstance(shdi, float)
+        self.assertLessEqual(shdi, 1.0)
+
+    # Note: This test can only be executed if the whole SHDI is in the database.
```

Comment:
would you leave this here or rather remove?

Reply:
I want to leave it to make clear that this behavior has been tested.